java/code/src/com/suse/manager/webui/utils/gson/MaintenanceCalendarJson.java

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright (c) 2020 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.suse.manager.webui.utils.gson;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * JSON representation of the Maintenance Window scheduling
+ */
+public class MaintenanceCalendarJson {
+
+    /** schedule ID */
+    private Long scheduleId;
+
+    /** calendar ID */
+    private Long calendarId;
```

Comment:
Since we can remove the `scheduleId` here the only param left that deals with schedues is `scheduleNames. I will have to adapt the names for the frontend as well, but I don't think it would cause any issues.